tox.ini

```diff
@@ -48,10 +48,13 @@ commands =
     py.test --create-db -n 2 -m 'not es_tests' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/editors/ --ignore src/olympia/amo/ --ignore src/olympia/users/ {posargs}
 
 [testenv:ui-tests]
-passenv = DISPLAY
+passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL ELASTICSEARCH_LOCATION
 install_command = pip install {packages}
 deps = -rtests/ui/requirements.txt
-commands = pytest --verify-base-url --driver Firefox tests/ui {posargs}
+commands =
+    make update_deps
+    python manage.py waffle_switch --create super-create-accounts on
+    py.test --driver Firefox -n 2 -m 'not es_tests' -v tests/ui {posargs}
```

Comment:
Note that by removing `--verify-base-url` all tests will attempt to run even if the application does not return a 200 OK. Also, do we need the `-m` if the test path does not contain tests with the marker we're filtering on?